healthScore/healthScore/views.py

```diff
@@ -16,43 +17,94 @@ def test_default_values(request):
     # h = hospital.objects.create(name="NYU", address="246", email="nyu@nyu.com", password="123435", contactInfo="123456781")
     return HttpResponse("<h1>Finally Workingggggggg. Welcome to HealthScore</h1>")
 
+def view_health_history(request):
+    # Filtering to just userID=5 to simulate it being a users view.
+    history_list = healthRecord.objects.filter(userID=5)
```

Comment:
We could add a condition before the start of the logic just to ensure that the right kind of method is used to hit this endpoint. 
Since this is a view health history, we could add a 
if request.method == "GET" : 

and then add the rest of the logic

Reply:
Added